igdiscover/Snakefile

```diff
@@ -653,6 +653,8 @@ def germlinefilter_criteria(wildcards, input):
 		criteria += ['--allow-stop']
 	# if conf['allow_chimeras']:
 	#	criteria += ['--allow-chimeras']
+	if 'max_db_diff' in conf:
```

Comment:
As soon as you add the new setting also to the `Config` class, this will always be true. `Config` always includes all the configuration settings. If a setting wasn’t given in the `igdiscover.yaml` file, a default value is used.

Reply:
> The test fails because you need to add the configuration option also to `config.py`.

Added option to config.py. Strange it run fine locally without this.

> The commit message should not include the `* igdiscover/germlinefilter.py` part. The `germlinefilter:` is fine, but repeating the name of the tool wastes precious characters. Better spend them on describing what the commit is about.

I force pushed updated commit removing full path from commit message.
However commit message changed to be more general.

Reply:
> I force pushed updated commit removing full path from commit message.
> However commit message changed to be more general.

Hm, you need to figure this out because I will complain about this every time. Do you use your editor to make commits? There must be a setting that disables this behavior.